File: problems/algorithms/function-sum/php/code/spec/sum.spec.php
<?php
require __DIR__.'/../src/sum.php';

describe('Number Tools', function() {

  it('adding 1 + 2', function() {
    expect(sum(1, 2))->toBe(3);
  });

  xit('adding 3 + 2', function() {
    expect(sum(3, 2))->toBe(5);
  });

});
?>

File: problems/algorithms/birthday-cake-candles/ruby/code/spec/sum.spec.rb
require_relative "../src/sum"

describe Sum do
  describe ".compute" do
    context "adding 1 + 2" do
      it "return 3" do
        expect(Sum.compute(1, 2)).to eql(3)
      end
    end
  
    context "adding 3 + 2" do
      it "return 5" do
        expect(Sum.compute(3, 2)).to eql(5)
      end
    end
  end
end
 